main.go

```diff
@@ -28,22 +29,37 @@ var (
 	leaserSyncInterval            time.Duration
 	ipLeaseTime                   = os.Getenv("WGS_IP_LEASE_TIME")
 	leasesFilename                = os.Getenv("WGS_IP_LEASEs_FILENAME")
-	flagSet                       = flag.NewFlagSet("", flag.ExitOnError)
-	flagConfig                    = flagSet.String("config", "", "(Required for agent) Path of the config file")
+
+	flagAgent   = flag.Bool("agent", false, "Run application in \"agent\" mode")
+	flagConfig  = flag.String("config", "", "Config file (only used in agent mode)")
+	flagServer  = flag.Bool("server", false, "Run application in \"server\" mode")
+	flagVersion = flag.Bool("version", false, "Prints out application version")
 )
 
 func main() {
+	flag.Parse()
+
 	if len(os.Args) < 2 {
-		log.Fatalln(usage)
+		flag.PrintDefaults()
```

Comment:
```suggestion
		flag.PrintDefaults()
		return
```

Currently double prints with no args:
```
$ ./wiresteward
  -agent
        Run application in "agent" mode
  -config string
        Config file (only used in agent mode)
  -server
        Run application in "server" mode
  -version
        Prints out application version
  -agent
        Run application in "agent" mode
  -config string
        Config file (only used in agent mode)
  -server
        Run application in "server" mode
  -version
        Prints out application version
$ ./wiresteward foobar
  -agent
        Run application in "agent" mode
  -config string
        Config file (only used in agent mode)
  -server
        Run application in "server" mode
  -version
        Prints out application version